packages/material-ui-codemod/src/v5.0.0/jss-to-styled.js

```diff
@@ -43,8 +48,7 @@ export default function transformer(file, api, options) {
     }
 
     if (!prefix) {
-      // 4. use file name
```

Comment:
We should avoid comments that just repeat the code. This probably wasn't a separate function when adding the comment. Then it did make sense to add a comment (here we're just using comments for extracting the code into a separate function). Now it was just noise.